compiler-plugin/src/main/kotlin/com/google/devtools/ksp/symbol/impl/kotlin/KSTypeReferenceImpl.kt

```diff
@@ -39,6 +39,29 @@ class KSTypeReferenceImpl private constructor(val ktTypeReference: KtTypeReferen
     override val location: Location by lazy {
         ktTypeReference.toLocation()
     }
+    override val parent: KSNode? by lazy {
+        var parentPsi = ktTypeReference.parent
+        while (parentPsi != null && parentPsi !is KtAnnotationEntry && parentPsi !is KtFunctionType &&
```

Comment:
Can you indent the while so that it's easy to visually compare with the when below?